### PR TITLE
WIP add cd and prox grad solvers

### PR DESCRIPTION
**Pull request by mathurinm**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Install
+--------
+
+This benchmark can be run using the following commands:
+
+.. code-block::
+
+   $ pip install -U benchopt
+   $ git clone https://github.com/benchopt/benchmark_lasso
```
Comment (agramfort) [inline README.rst]:
```suggestion
   $ git clone https://github.com/benchopt/benchmark_mcp
```

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+   $ benchopt run ./benchmark_lasso
+
+Apart from the problem, options can be passed to `benchopt run`, to restrict the benchmarks to some solvers or datasets, e.g.:
+
+.. code-block::
+
+	$ benchopt run ./benchmark_lasso -s sklearn -d boston --max-runs 10 --n-repetitions 10
```
Comment (agramfort) [inline README.rst]:
```suggestion
	$ benchopt run ./benchmark_mcp -s sklearn -d boston --max-runs 10 --n-repetitions 10
```

Comment (agramfort) [reply README.rst]: there is no skelarn actually :)

Comment (josephsalmon) [reply README.rst]: readme is better now. Thanks for the update.

**Comment by agramfort** [comment]
@josephsalmon can you setup the CI?

**Comment by josephsalmon** [comment]
could you tell me how to do that?

**Comment by agramfort** [comment]
go here: https://github.com/benchopt/benchmark_mcp/actions/new

then copy and adjust the files from
https://github.com/benchopt/benchmark_logreg_l2/tree/master/.github/workflows

>

**Comment by josephsalmon** [comment]
I did it on master apparently, was it what you had in mind?

**Comment by agramfort** [comment]
you need to update your branch and push to see the CI starting

>

**Comment by agramfort** [comment]
green !

**Pull request merged**